src/components/reports/reportSummary/reportSummaryDetails.scss

```diff
@@ -0,0 +1,34 @@
+@import url("~@patternfly/patternfly/base/patternfly-variables.css");
+
+.reportSummaryDetails {
+    margin-bottom: var(--pf-global--spacer--md);
+    display: flex;
+    align-Items: flex-end;
+}
+
+.text {
+  padding-bottom: var(--pf-global--spacer--sm);
+  line-height: var(--pf-global--LineHeight--sm);
+  font-size: var(--pf-global--icon--FontSize--xs);
+}
+
+.units {
+  padding-left: var(--pf-global--spacer--xs);
+  padding-bottom: var(--pf-global--spacer--sm);
+  line-height: var(--pf-global--LineHeight--sm);
+  font-size: var(--pf-global--icon--FontSize--xs);
```

Comment:
Discovered that these font styles are not valid. Our hero numbers are no longer styled.